cooperatives/dcrat

```diff
@@ -0,0 +1,23 @@
+---
+title: Digital Cycle Audit Tool (DCRAT)
+parent: Cooperatives
+---
+
+# Digital Cycle Audit Tool (DCRAT)
+
+|                   |                                          |
+|:------------------|:-----------------------------------------|
+| model             | Collaborative Independent Development
+| service type      | Transportation
+| country           | United Kingdom
+| government members | Oxfordshire County
+| government type   | municipal
+| license           | unknown
+| website           | [https://drive.google.com/file/d/1dozE-acykNmRWAzMGh_XLbUMZodgJUTT/view] (https://drive.google.com/file/d/1dozE-acykNmRWAzMGh_XLbUMZodgJUTT/view)
```

Comment:
Based on the description, it seems this may be dissolved or have a defunct status now. Could you look into this and try to figure out if this is still running, and add a status row here with your findings? @abbeyns149

Reply:
It looks like you're right - this was just a yearlong project but now it is inactive. I will put a row to reflect that!